### PR TITLE
feat: allow users to overwrite 'host' and 'explorer' in network settings

### PR DESCRIPTION
**Pull request by guanqun**
### What I did
I'd like to have a setting for mainnet. Infura is not the only option here. I can specify "host" with another eth RPC provider.
similar to etherscan, I'd like to have another url, as the original one is blocked here.

### How I did it
I add the code in _config.py

### Checklist

- [ ] I have confirmed that my PR passes all linting checks
- [ ] I have included test cases
- [ ] I have updated the documentation
- [ ] I have added an entry to the changelog

### CODE REVIEW

**Comment by skellet0r** [comment]
@guanqun I'm sure the network docs need to shed some more light on this, but you can actually overwrite network settings already.
There are a few routes one can take:
1. Via the command line, you can run `brownie networks modify`
2. Adding a network-config.yaml in your repository, and then importing via `brownie networks import` (you'll have to set the overwrite flag to `True`)
3. Lastly, going directly to the `~/.brownie/network-config.yaml` file and manually correcting the entries of interest.

The simplest option (and most future proof) is option 2. Just remember when committing your network-config, not to include and secrets.

[Example of a network-config.yaml](https://github.com/eth-brownie/brownie/blob/master/brownie/data/network-config.yaml)

> tl;dr: run `brownie networks -h` and you'll see a bunch of available options for managing your available networks

**Comment by guanqun** [comment]
Thank you @skellet0r , I wasn't aware of this usage. I'll check this out.
